_includes/payment-order-other-features.md

```diff
@@ -341,19 +351,19 @@ A list of possible operations and their explanation is given below.
             "rel": "view-paymentorder",
             "contentType": "application/javascript"
         },
-        {
+        {   `DEPRECATED DO NOT USE!`
             "method": "POST",
             "href": "{{ page.api_url }}/psp/paymentorders/{{ page.payment_order_id }}/captures",
             "rel": "create-paymentorder-capture",
             "contentType": "application/json"
         },
-        {
+        {   `DEPRECATED DO NOT USE!`
```

Comment:
`create-paymentorder-cancel` is not deprecated. It is `create-cancel` (which is missing from the operations list) that is deprecated.

```suggestion
        {
```